src/CommunityToolkit/Xamarin.CommunityToolkit/Views/Snackbar/Helpers/iOS/IOSSnackBar.ios.cs

```diff
@@ -17,7 +17,7 @@ class IOSSnackBar
 
 		public string ActionButtonText { get; protected set; }
 
-		public SnackBarAppearance Appearance { get; } = new SnackBarAppearance();
+		public SnackBarAppearance Appearance { get; protected set; } = new SnackBarAppearance();
```

Comment:
Any reason to don't make this `set` public? That way we don't need the `SetMethod`

Reply:
1. Following the style of other properties. 
2. This class is for internal usage. We have public SnackbarOptions.